testing/ef-tests/src/result.rs

```diff
@@ -24,6 +27,8 @@ pub enum Error {
     BlockProcessingFailed {
         /// The block number for the block that failed
         block_number: u64,
+        /// The partial program inputs up to and including the block that failed
+        partial_program_inputs: Vec<(RecoveredBlock<Block>, ExecutionWitness)>,
```

Comment:
what does partial mean here?

I find the term program a bit confusing here, what is the program here?

Reply:
fine with the partial_program_inputs, but additional context that explains what this means would be helpful